source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -0,0 +1,321 @@
+
+Below is the same example as above with the OAuth Token passed in via the request headers:
+
+.. code-block:: bash
+
+  curl -k -H "Content-Type: application/json" -H "Authorization: <YOUR OAUTH TOKEN>" -X POST -d '{"call":{"external_call_unique_id":"123ABC","start_time":"2016-08-08 11:03:31 -0700","destination_phone_number":9093900003,"calling_phone_number":8779257383,"advertiser_campaign_id_from_network":85,"call_direction":"inbound","recording_url":"<CALL RECORDING URL>"},"signals":[{"name":"sale","value":1},{"name":"quote","value":1}],"custom_data":[{"name":"channel","value":"Paid Search"},{"name":"line_of_business","value":"Social"}]}'  https://invoca.net/api/@@CALL_INGESTION_API_VERSION/calls.json
+
+
+
+Errors
+------
+
+The Call Ingestion API clearly identifies errors when a request cannot be processed.
+
+**Invalid Inputs**
+
+If invalid parameters are passed, an error will be returned with a 403 response code.
+
+For example, if a **call** or parameters within the call are not passed in the request, the following error will be returned.  
+If there are multiple issues with the request, we will do our best to package all of the issues together in one response message.
+
+**Response (403 Forbidden):**
+
+.. code-block:: json
+
+  {
+    "errors": {
+      "class": "RecordInvalid",
+      "invalid_data": "Validation failed: 'call' is required"
+    }
+  }
+
+-----
+
+**Permission Errors**
+
+If you do not have access to the Call Ingestion API, the following error will be returned with a 403 response code.  
+*Please note that the Call Ingestion API is enabled per network.  Please contact the Invoca support team at questions@invoca.com for setup assistance.*
+
+**Response (403 Forbidden):**
+
+.. code-block:: json
+
+    {
+      "errors": {
+        "class": "UnauthorizedOperation",
+        "invalid_data": "You do not have permissions to perform the requested operation."
+      }
+    }
+
+-----
+
+**Authorization Errors**
+
+If you do not have access to the **advertiser_campaign_id_from_network** an error will be returned with a 403 response code.
+For example, if you pass an **advertiser_campaign_id_from_network** that you do not have access to, the following error will be returned.
+
+**Response (403 Forbidden):**
+
+.. code-block:: json
+
+    {
+      "errors": {
+        "class": "UnauthorizedAdvertiser",
+        "invalid_data": "You do not have access to this advertiser"
+      }
+    }
+-----
+
+**Campaign Configuration Related Errors**
+
+In order to fully utilize the Call Ingestion API, there are some configuration requirements for the campaign that the call is being submitted under.  Here's a list of those requirements:
+  * Campaigns must be setup with a campaign type of **External**
+  * Campaigns need to be configured with transciption enabled
+  * Campaigns need to have Signal AI enabled at at least one Signal configured
```

Comment:
It looks like "at at" should be "and at".

Also, currently the way we operate is that if "signals_ai" is enabled, we will transcribe regardless of whether there is any associated signal. On the other hand if "signals_voice" is enabled, we do require associated signals in order to transcribe. I'm not saying we need to get into this detail here, but it may be confusing to users.

Reply:
Would it make sense to say "Campaigns need to either have Signal AI enabled or at least one Voice Signal configured"?

Reply:
Thanks @leekirchhoff @kamerynn - updates pushed.